pages/services/confluent-zookeeper/2.5.0-4.1.3e/release-notes/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Discover the new features, updates, and known limitations in this release of the Confluent ZooKeeper Service
+title: Release Notes
+menuWeight: 120
+model: /services/kafka-zookeeper/data.yml
+render: mustache
+---
+
+# Release Notes for Confluent ZooKeeper Service version 2.5.0-4.1.3e 
+
+- Update to Confluent Zookeeper version 4.1.3 
```

Comment:
@rathohit311356 let's add the SDK bump to `0.55.4` also here

Reply:
Done